### ISSUE TITLE
The amadeus api failed to fetch the data as it does not have ok HTTP data. 

### ISSUE THREAD

**Issue by staceyJayneLewis** [staceyJayneLewis/holiday-hotspots#8]
As you can see below is the error message dispplayed:
![image](https://github.com/staceyJayneLewis/holiday-hotspots/assets/126517499/0c643483-7690-424e-a56d-123f52fb63b7)

**Comment by staceyJayneLewis** [comment]
To solve this issue I looked into the API documentation and realised that there were certain restrictions on the api link I was using to retrieve the locations. Alternatively I found a different link for points of interest that I can use instead where I can make up to 400 call requests for free. You will see in this screenshot below that I changed the amadeus API link to another suggested link which say's 'poi' for points of interest which seems to work well for the free cities data provided with the test API. 
![image](https://github.com/staceyJayneLewis/holiday-hotspots/assets/126517499/ad6658ca-bf77-46a4-a37b-aaefbbf9599f)
